staff/victor-abadia/cv/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+
+<body>
+    <header>
+        <h1>Eddard Stark</h1>
+    </header>
+    <br>
+    <nav>
+        <a href="#personal"><h4>Personal Data</h4></a>
+        <a href="#education"><h4>Education</h4></a>
+        <a href="#employment"><h4>Employment</h4></a>
+    </nav>
+    <br>
+    <section>
```

Comment:
I would change this `<section>` tag for a `<main>` tag

Reply:
here the link: https://developer.mozilla.org/es/docs/Web/HTML/Elemento/main